extensions/pointcloud/example-alaska.json

```diff
@@ -0,0 +1,261 @@
+{
+  "assets": {},
+  "bbox": {
+    "maxx": -149.2527639,
+    "maxy": 61.78936655,
+    "maxz": 1981.200833,
+    "minx": -149.3006585,
+    "miny": 61.76668361,
+    "minz": 11.48480563
+  },
+  "geometry": {
+    "coordinates": [
+      [
+        [
+          [
+            -149.30045496703195,
+            61.76682620765713
+          ],
+          [
+            -149.25327277445712,
+            61.76657763000873
+          ],
+          [
+            -149.25272374552367,
+            61.78921108684322
+          ],
+          [
+            -149.30023535632952,
+            61.7894132390075
+          ],
+          [
+            -149.30080489041393,
+            61.766924315502195
+          ],
+          [
+            -149.30045496703195,
+            61.76682620765713
+          ]
+        ]
+      ]
+    ],
+    "type": "MultiPolygon"
+  },
+  "id": "AK_MatanuskaSusitna-Lot2_2011_000227.laz",
+  "properties": {
+    "c:description": "USGS 3DEP LiDAR",
```

Comment:
c:id and c:description are given, but no collection is linked to. Would remove both properties.